### PR TITLE
Enable session storage experiment, fix E2E tests

### PR DESCRIPTION
**Pull request by kamilszostak**

### CODE REVIEW

File: JavaScript/JavaScriptSDK.Tests/E2ETests/snippet.tests.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ class SnippetTests extends TestClass {
                     Assert.deepEqual("test", pv.url, "url was set correctly");
                     Assert.deepEqual({ property: "p1" }, pv.properties, "properties were set correctly");
                     Assert.deepEqual({ measurement: 5 }, pv.measurements, "measurements were set correctly");
-                    
+
```
Comment (dnduffy) [inline JavaScript/JavaScriptSDK.Tests/E2ETests/snippet.tests.ts]: nitpick: You could just delete the blank lines.

File: changeSdkVersion.ps1 [hunk 2]
```diff
@@ -26,7 +26,7 @@ if (-Not ($version -match "\d+\.\d+\.\d+")) {
 $packagesJson.version = $version
 $packagesJson = $packagesJson | ConvertTo-Json 
 $packagesJson = $packagesJson -replace "\\u0026", "&"
-$packagesJson | Out-File $packageJsonPath
+[System.IO.File]::WriteAllLines($packageJsonPath, $packagesJson)
```
Comment (dnduffy) [inline changeSdkVersion.ps1]: PowerShell has a built in JSON support so why not use it?

**Comment by dnduffy** [comment]
I'm curious about the file output changes.
:shipit:

File: changeSdkVersion.ps1 (hunk 2 above)
Comment (kamilszostak) [inline changeSdkVersion.ps1]: Files are incorrectly encoded and GitHub is showing them as binary. WriteAllLines will save them as UTF-8-BOM.

File: JavaScript/JavaScriptSDK.Tests/E2ETests/snippet.tests.ts (hunk 1 above)
Comment (kamilszostak) [inline JavaScript/JavaScriptSDK.Tests/E2ETests/snippet.tests.ts]: Removed :smile:

**Comment by MaxShehovtsov** [comment]
:shipit:

**Pull request merged**